proto/substrait/algebra.proto

```diff
@@ -433,6 +433,7 @@ message Rel {
     CrossRel cross = 12;
     //Physical relations
     HashJoinRel hash_join = 13;
+    NestedLoopJoinRel nested_loop_join = 18;
```

Comment:
Given that this is the third defined join relation perhaps it should come after MergeJoinRel (here in Rel and as defined later)?